src/thirty_two.rs

```diff
@@ -1,5 +1,5 @@
-use crate::TransmutingByteSlices;
-use core::{cmp, hash::Hasher};
+use crate::{TransmutingByteSlices, UnalignedBuffer};
+use core::{cmp, hash::Hasher, iter::once};
```

Comment:
Reminder about the free functions; I prefer this to be called as `iter::once`, so we just import `iter`.